index.js

```diff
@@ -9,14 +9,13 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
     if (process.env.EMBER_CLI_FASTBOOT !== 'true') {
+      // In nested addons, app.bowerDirectory might not be available
+      var bowerDirectory = app.bowerDirectory || 'bower_components';
+      // In ember-cli < 2.7, this.import is not available, so fall back to use app.import
+      var importShim = typeof this.import !== 'undefined' ? this : app;
```

Comment:
Good point, we can probably move toward that, I'll add a note about polyfill in the readme for ember-cli<2.7.